validation/POIs_validation.py

```diff
@@ -0,0 +1,71 @@
+import geopandas as gpd
+import numpy as np
+import matplotlib.pyplot as plt
+from geopy.distance import great_circle
+
+official_filename = "Restaurant_and_Food_Related.shp"
+OSM_filename = "belfast_gb_2019_1600m_buffer.gpkg"
+POIs_name = "fresh_food_market"
+
+gdf = gpd.read_file(OSM_filename, layer='destinations')
+OSM_points = gdf[gdf['dest_name']==POIs_name]
+
+OPoints = gpd.GeoDataFrame.from_file(official_filename)

```

Comment:
Can you give your variables more intuitive names? like gdf_osm and gdf_official? Easier to follow along...

Reply:
Changes made